requirements.txt

```diff
@@ -1,11 +1,13 @@
 numpy==1.26.0; platform_machine != 'armv7l'
 numpy==1.25.2; platform_machine == 'armv7l'
-pandas==2.0.3
+pandas==2.1.1
 pandas-ta==0.3.14b
 
 ccxt==4.1.8
 cryptography==41.0.4
 aiohttp==3.8.6
+# for increasing pandas performance
+Bottleneck==1.3.7
```

Comment:
I don't see this used.

We'll not force people to  install random, unused dependencies.

Reply:
https://pandas.pydata.org/docs/getting_started/install.html#performance-dependencies-recommended

Reply:
and that's self-importing, or imported by pandas ... ? 

looking at the bottleneck documentation only shows explicit samples, but doesn't seem to mention that it's auto-used by pandas ?

Reply:
as per pandas 2.1.1 documentation: 
`compute.use_bottleneck : bool
Use the bottleneck library to accelerate if it is installed, the default is True Valid values: False,True [default: True] [currently: True]`
[reference](https://pandas.pydata.org/docs/reference/api/pandas.set_option.html#pandas.set_option)